navbar-animation/README.md

```diff
@@ -0,0 +1,38 @@
+# create-svelte
```

Comment:
Can you edit the read-me to explain what this example shows?

Reply:
Done